lib/cli/index.js

```diff
@@ -1,8 +1,16 @@
 'use strict';
 
+var roadrunner = require('roadrunner');
+roadrunner.load();
```

Comment:
we may need a specific a name here: https://github.com/kittens/roadrunner/blob/master/index.js#L28-L29